docs/install/README.md

```diff
@@ -899,7 +899,7 @@ Stargate node.
 
 ```sh
 cat <<EOF | kubectl -n k8ssandra-operator apply -f -
-apiVersion: k8ssandra.io/v1alpha1
+apiVersion: k8ssandra.io/v1alphav1.0.0-alpha.11
```

Comment:
I think you've got an extra 1 in there... Unless we really did a lot of work today to get to 11 😉